src/domain/estimations/entities/schemas/estimation.schema.ts

```diff
@@ -0,0 +1,10 @@
+import { Schema } from 'ajv';
+
+export const estimationSchema: Schema = {
```

Comment:
nitpick: We can use `JSONSchemaType<Estimation>` here so that we have some extra TS validation in-place

Reply:
Changed in [cc1b105](https://github.com/5afe/safe-client-gateway-nest/pull/269/commits/cc1b10550ff69e6516e4a0956cc3820a7d6f5c43)